pom.xml

```diff
@@ -53,7 +53,7 @@
     <dependency.high-scale-lib.revision>1.1.2</dependency.high-scale-lib.revision>
     <mondrian.version>10.0.0.0-SNAPSHOT</mondrian.version>
     <oss-licenses.version>10.0.0.0-SNAPSHOT</oss-licenses.version>
-    <pdi.version>10.0.0.0-SNAPSHOT</pdi.version>
+    <pdi.version>10.1.0.0-SNAPSHOT</pdi.version>
```

Comment:
I don't think you want to do this on the 10.0 branch

Reply:
@peterrinehart thanks for reviewing the changes.
changed the version to 10.0 please, build is successful.